pages/common/sdk.md

```diff
@@ -4,34 +4,34 @@
 > Supports Java, Groovy, Scala, Kotlin, Gradle, Maven, Vert.x and many others.
 > More information: <https://sdkman.io/usage>.
 
-- Install a specific version of Gradle:
+- Install a SDK version:
```

Comment:
```suggestion
- Install an SDK version:
```
(https://oneminuteenglish.org/en/a-or-an-before-acronyms/)

But I think even better would be
```suggestion
- Install a specific SDK version:
```